app/demo_data/fake_student.rb

```diff
@@ -218,11 +218,11 @@ def add_attendance_events
 
   def add_discipline_incidents
     d = {
-      0 => 0.83,
-      1 => 0.10,
-      2 => 0.03,
-      (3..5) => 0.03,
-      (6..15) => 0.01,
+      0 => 0.23,
+      1 => 0.30,
+      2 => 0.23,
+      (3..5) => 0.13,
+      (6..15) => 0.11,
```

Comment:
Can you say more about this?

Reply:
@kevinrobinson I meant to back this out after seeding my local db, but it might be worth keeping. With the default distribution, we don't have enough fake students to to get any discipline incidents in the dashboard, so the demo site will just look blank.

Reply:
Ah got it.  Reasonably realistic distributions are good, but development data that is actually helpful for development is the most important :)